js/controller.js

```diff
@@ -296,7 +297,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         this.state.mainVideoInnerWrapper.append("<div class='oo-player-skin'></div>")
       }
 
-      this.state.mainVideoInnerWrapper.attr('style', '');
```

Comment:
We should keep this line where `style` is cleared. This could've been added for a reason.

Reply:
This line was added by me in one of the past PRs. In mainVideoInnerWrapper, style = "width: inherit; height: inherit; position: relative; padding-top: 50%;". Inline-style is difficult to override so I changed style = ''. Now the style changes are only for video360. For the flat video, everything remains unchanged